entrypoint.sh

```diff
@@ -33,4 +33,8 @@ else
     echo "No extra plugin packages provided. Specify 'OTEAPI_PLUGIN_PACKAGES' to specify plugin packages."
 fi
 
-hypercorn asgi:app --bind 0.0.0.0:8080 "$@"
+if [ "$DEV_ENV" = "1" ]; then
+    python -m debugpy --wait-for-client --listen 0.0.0.0:5678 -m hypercorn --bind 0.0.0.0:8080 --reload asgi:app "$@"
```

Comment:
Just for.. uhm.. "fun".. could you try the following:
```suggestion
    python -m debugpy --wait-for-client --listen 0.0.0.0:5678 -m hypercorn --bind 0.0.0.0:8080 "$@" asgi:app
```

And if that works and there's no complain about `--reload` not being present, then we can change the line below accordingly.